src/colorizer/utils/url_utils.ts

```diff
@@ -14,6 +14,12 @@ const URL_PARAM_RANGE = "range";
 const URL_PARAM_COLOR_RAMP = "color";
 const URL_COLOR_RAMP_REVERSED_SUFFIX = "!";
 
+const ALLEN_FILE_PREFIX = "/allen/";
+const ALLEN_PREFIX_TO_HTTPS: Record<string, string> = {
+  "/allen/aics/assay-dev": "https://dev-aics-dtp-001.int.allencell.org/assay-dev",
+  "/allen/aics/microscopy": "https://dev-aics-dtp-001.int.allencell.org/microscopy",
+};
```

Comment:
I think I would check for `\\allen` and `//allen` as well.  Sometimes people paste in windows paths with backslashes, and sometimes the leading slash is doubled.  So maybe normalize the slashes to forward slash first, if the first character is a slash?

Reply:
(in the convert function, that is)